.travis.yml

```diff
@@ -7,46 +7,52 @@ matrix:
     - os: linux
       language: python
       python: "3.6"
-#cache:
-#  apt: true
-#  directories:
-#    - $HOME/miniconda
-services:
-  - xvfb
+
+env:
+  global:
+    - CC_TEST_REPORTER_ID=3865090108d29191b1d34760d2ed5825d67b7566f0f87090e688b7513fdef063
+
 before_install:
-  # install miniconda for Python dependencies
+  # setup miniconda
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
     else
       wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh;
     fi
-  - bash miniconda.sh -b -p $HOME/miniconda -f
-  - source "$HOME/miniconda/etc/profile.d/conda.sh"
-  - hash -r
-  - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b -p $HOME/miniconda -f
+  - export PATH=$HOME/miniconda/bin:$PATH
+  - conda config --set always_yes yes 
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then ulimit -S -n 4096; ulimit -a; fi
+
+install:
   - conda config --add channels defaults
   - conda config --add channels bioconda
   - conda config --add channels conda-forge
-  - conda info -a
-  # if matplotlib functionality is tested
-  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then ( sudo Xvfb :99 -ac -screen 0 1024x768x8; echo ok )& fi
-  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then ulimit -S -n 4096; ulimit -a; fi
-install:
-  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then
-      conda env create -q -f conda_env.osx.txt -n gimme;
-    else
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       conda env create -q -f conda_env.dev.txt -n gimme;
+    else
+      conda env create -q -f conda_env.osx.txt -n gimme;
     fi
-  - conda activate gimme
+  - source activate gimme
```

Comment:
Why `source activate` (older) instead of `conda activate` (newer)?

Reply:
Because conda has just been installed (`conda activate` would work in a new terminal).

Testing this locally, I have no idea if (or how) it managed to work before.